packages/squiggle-lang/src/fr/system.ts

```diff
@@ -0,0 +1,32 @@
+import { makeDefinition } from "../library/registry/fnDefinition.js";
+import { frNumber } from "../library/registry/frTypes.js";
+import { FnFactory } from "../library/registry/helpers.js";
+import { ImmutableMap } from "../utility/immutableMap.js";
+import { Value, vString } from "../value/index.js";
+
+// automatically updated on release by ops/ patch-js utils
+const VERSION = "0.9.3-0";
+export function makeSystemConstants(): ImmutableMap<string, Value> {
+  return ImmutableMap([["System.version", vString(VERSION)]]);
+}
+
+const maker = new FnFactory({
+  nameSpace: "System",
+  requiresNamespace: true,
+});
+
+export const library = [
+  // It might make sense to later make this a constant, as this number shouldn't change at runtime.
```

Comment:
Yes, but you can't just add it to `stdlib`, which is cached in `SqProject`.

I guess you could convert `getStdLib()` to `getStdLib(env: Environment)`, and then invalidate it in `SqProject` on `setEnvironment`...